postBuild

```diff
@@ -0,0 +1 @@
+jupyter labextension install @jupyter-widgets/jupyterlab-manager
```

Comment:
I'd prefer not to pollute the root directory with more files. Can't the tool that needs this read the file out of a hidden directory?

Answered my own question, yes - repo2docker looks in `.binder` for files. https://github.com/jupyterhub/repo2docker/blob/0dba6111c6b1f0a5f0208193ca9b16891d4469e6/repo2docker/buildpacks/base.py#L485-L505
This isn't prominent in the documentation. I think `environment.yml` can go there too.